out_writeapi.go

```diff
@@ -55,10 +57,11 @@ var (
 )
 
 const (
-	chunkSizeLimit      = 9 * 1024 * 1024
-	queueRequestDefault = 1000
-	queueByteDefault    = 100 * 1024 * 1024
-	exactlyOnceDefault  = false
+	chunkSizeLimit             = 9 * 1024 * 1024
+	queueRequestDefault        = 1000
+	queueByteDefault           = 100 * 1024 * 1024
+	exactlyOnceDefault         = false
+	queueRequestScalingPercent = 0.1
```

Comment:
I suggest we set this to our desired production value of 0.8. However, when testing you can set it lower on your local machine. The reason for this is so that we don't forget to set it to the target production value.

Reply:
Done.